.vsts-pipelines/builds/ci-internal.yml

```diff
@@ -10,7 +10,7 @@ resources:
     ref: refs/heads/release/2.2
 
 phases:
-- template: .vsts-pipelines/templates/phases/default-build.yml@buildtools
+- template: .azure/templates/jobs/default-build.yml@buildtools
```

Comment:
It looks like the macOS, Windows, and Linux steps are all the same. You could reduce this to one import like this
```yml
jobs:
- template: .azure/templates/project-ci.yml@buildtools
  parameters:
    beforeBuild:
     - task: NodeTool@0
       displayName: Use Node 8.x
       inputs:
         versionSpec: 8.x
```